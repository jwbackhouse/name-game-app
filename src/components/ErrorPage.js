import React from 'react';
import { Link } from 'react-router-dom';


const ErrorPage = () => (
  <div>
    <p>Doh! 404</p>
    <Link to='/'>Go home</Link>
  </div>
);

export default ErrorPage;