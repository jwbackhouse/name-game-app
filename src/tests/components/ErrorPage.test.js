import React from 'react';
import { shallow } from 'enzyme';
import ErrorPage from '../../components/ErrorPage';

test('Should render ErrorPage', () => {
  const wrapper = shallow(<ErrorPage />);
  expect(wrapper).toMatchSnapshot();
})