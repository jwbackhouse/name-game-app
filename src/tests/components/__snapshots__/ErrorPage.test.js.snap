// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ErrorPage 1`] = `
<div>
  <p>
    Doh! 404
  </p>
  <[object Object]
    to="/"
  >
    Go home
  </[object Object]>
</div>
`;
